README.md

```diff
@@ -35,7 +35,7 @@ helm install gatekeeper/gatekeeper  \
     --name-template=gatekeeper \
     --namespace gatekeeper-system --create-namespace \
     --set enableExternalData=true \
-    --set controllerManager.dnsPolicy=ClusterFirst,audit.dnsPolicy=ClusterFirst
+    --set validatingWebhookTimeoutSeconds=7
```

Comment:
could we add a comment to explain how we came up with 7?

Reply:
Added a comment to README below